### DISCUSSION TITLE
Connecting to MySQL database with UDS

### DISCUSSION THREAD

**Discussion by farooqkz** [TaKO8Ki/gobang#150] [Q&A] [answered]
Hello.
How can I connect to a MySQL database by providing path to a Unix Domain Socket?

**Comment by TaKO8Ki** [comment, marked as answer]
gobang uses [SQLx](https://github.com/launchbadge/sqlx) for executing SQL and it [supports unix domain sockets](https://docs.rs/sqlx/latest/sqlx/mysql/struct.MySqlConnectOptions.html#properties), but gobang doesn't. However it it easy to support unix domain in gobang. I'll add an issue about it.
